.gitignore

```diff
@@ -50,16 +47,19 @@ typings/
 # dotenv environment variables file
 .env
 
-# next.js build output
+# build output
+_agstate
 .next
-
-# bundles
+build
+bundles
```

Comment:
This conflicts with existing files in `packages/SwingSet/test/bundles`

Reply:
I've added an exception to the existing `packages/SwingSet/test/bundles/.gitignore`